core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala

```diff
@@ -3512,7 +3538,10 @@ class WorkspaceServiceSpec
         new WorkspaceDescription().id(deletingAzureWorkspace.workspaceIdAsUUID).stage(WorkspaceStageModel.MC_WORKSPACE),
         // no azureContext, should not be returned
         new WorkspaceDescription().id(readyAzureWorkspace.workspaceIdAsUUID).stage(WorkspaceStageModel.MC_WORKSPACE),
-        new WorkspaceDescription().id(googleWorkspace.workspaceIdAsUUID).gcpContext(new GcpContext())
+        new WorkspaceDescription()
+          .id(googleWorkspace.workspaceIdAsUUID)
+          .gcpContext(new GcpContext())
+          .stage(WorkspaceStageModel.RAWLS_WORKSPACE)
```

Comment:
Without a stage set, there is no matching case in [this method](https://github.com/broadinstitute/rawls/blob/f85710459f67412b52516960707c4e685c6b7f83/core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/AggregatedWorkspaceService.scala#L106), which is called from the `listWorkspaces` method when WSM returns a matching workspace for the Rawls workspace (before this code change, Rawls workspaces would short-circuit and not check if there was a matching WSM workspace).

I believe we should always expect a WSM workspace to have the stage set.